scriptworker/cot/verify.py

```diff
@@ -540,7 +551,7 @@ def find_task_dependencies(task, name, task_id):
         CoTError: on failure.
     """
     link = chain.get_link(task_id)
-    log.debug("Verifying {} is in {} cot artifacts...".format(path, task_id))
+    log.debug(" Verifying {} is in {} cot artifacts...".format(path, task_id))
```

Comment:
We're able to add a Formatter per *level* per logger? I was completely unaware :)
This is just me playing with text formatting to try to make these audit logs more readable.  I imagine there's a lot we can do with wording and formatting to improve this, but this was a quick first stab at it.